_pages/en_US/get-started-(old-3ds).txt

```diff
@@ -43,18 +43,18 @@ We believe SpotPass and StreetPass are also safe for use at this time.
     </tr>
   </thead>
   <tbody>
+  	<tr>
+      <td style="text-align: center; font-weight: bold;">11.4.0</td>
+      <td style="text-align: center; font-weight: bold;">11.14.0</td>
+      <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(safecerthax)">Installing boot9strap (safecerthax)</a></td>
+    </tr>
     <tr>
       <td style="text-align: center; font-weight: bold;">1.0.0</td>
       <td style="text-align: center; font-weight: bold;">11.3.0</td>
       <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(soundhax)">Installing boot9strap (Soundhax)</a></td>
-    </tr>
-	<tr>
-      <td style="text-align: center; font-weight: bold;">11.4.0</td>
-      <td style="text-align: center; font-weight: bold;">11.14.0</td>
-      <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(safecerthax)">Installing boot9strap (safecerthax)</a></td>
     </tr>
     <tr>
-      <td style="text-align: center; font-weight: bold;" colspan="2">ALL VERSIONS<br></td>
+      <td style="text-align: center; font-weight: bold;" colspan="2">ALL VERSIONS</td>
```

Comment:
This effectively reverses the table order. I dont see the necessity for this, and it is outside of the scope of this PR (not documented, not in line with the cfw check changes, just seems out of place here)